projects/br_scs/src/Mirza/SupplyChain/Main.hs

```diff
@@ -129,8 +129,7 @@ runProgram so@ServerOptionsSCS{initDB = True, dbPopulateInfo =Just _, brServiceI
   ctx <- initSCSContext so
   migrate ctx $ connectionStr so
   runDbPopulate so
-runProgram so@ServerOptionsSCS{initDB =False, dbPopulateInfo =Just _, brServiceInfo =Just __} = do
-  runDbPopulate so
+runProgram so@ServerOptionsSCS{initDB =False, dbPopulateInfo =Just _, brServiceInfo =Just __} = runDbPopulate so
```

Comment:
You shouldn't need to use `__`, multiple `_` can exist.

Reply:
Also, the polymorphic data type Ashley used in the BR (and you guys discussed) may also make this more type safe. It's a useful technique, assuming it makes sense here.

Reply:
Lol not sure how the redundant `_` got in.

Reply:
It might be worth making use of that technique in SCS, but I think it makes sense in a different PR. this was merely a style tweak and should not have gotten in this PR anyway.

Opened #428 to keep track of it.